src/modal/NewModuleNameModal.tsx

```diff
@@ -0,0 +1,143 @@
+/**
+ * @license
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @author lizlooney@google.com (Liz Looney)
+ */
+
+import React, { useState, useRef } from 'react';
+
+import {
+  Alert,
+  Button,
+  Flex,
+  Input,
+  Modal,
+  Typography,
+} from 'antd';
+import type { InputRef } from 'antd';
+
+import * as commonStorage from '../storage/common_storage';
+
+
+export const TITLE_NEW_MECHANISM = 'New Mechanism';
+export const TITLE_RENAME_MECHANISM = 'Rename Mechanism';
+export const TITLE_COPY_MECHANISM = 'Copy Mechanism';
+export const TITLE_NEW_OPMODE = 'New OpMode';
+export const TITLE_RENAME_OPMODE = 'Rename OpMode';
+export const TITLE_COPY_OPMODE = 'Copy OpMode';
+
+const DESCRIPTION = 'No spaces are allowed in the name. Each word in the name should start with a capital letter.';
+
+export const EXAMPLE_MECHANISM = 'For example, GamePieceShooter';
+export const EXAMPLE_OPMODE = 'For example, AutoParkAndShoot';
```

Comment:
Should this be: 
For example: AutoParkAndShoot

Reply:
Done.